net.oz9aec.Gpredict.json

```diff
@@ -0,0 +1,49 @@
+{
+  "app-id": "net.oz9aec.Gpredict",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.28",
+  "sdk": "org.gnome.Sdk",
+  "command": "gpredict",
+  "finish-args": [
+    "--share=ipc", "--socket=x11",
+    "--socket=wayland",
+    "--share=network", "--filesystem=home"
```

Comment:
Does it really need full home access? If it is for the _~./gpredict_ directory, you can use `--persist=.gpredict` for that.

Reply:
Some users might like to use custom data files from their home directory, but this is generally atypical usage. Application data is stored under `.config/Gpredict`, so I'll restrict it to there.